tests/layer_tests/tensorflow_tests/test_tf_AddN.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+from common.utils.tf_utils import permute_nchw_to_nhwc
+
+# Testing operation AddN
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/AddN
+
+class TestAddN(CommonTFLayerTest):
+    # Overload inputs generation to fill dummy input
+    def _prepare_input(self, inputs_dict):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.random.randint(-3, 3, inputs_dict[input])
+        return inputs_dict
+    # input_list - should be an array, could be a single shape, or array of n-dimentional shapes
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_addn_placeholder_const_net(self, input_list, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->AddN      =>       Placeholder->AddN
+                         /                               /
+            Const-------/                   Const-------/
+
+        """
+
+        if len(input_list) == 0:
+            raise RuntimeError("Input list couldn't be empty")
+
+        is_list_of_shapes = isinstance(input_list[0], list)
+
+        import tensorflow as tf
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_inputs = []
+
+            if is_list_of_shapes:
+                for item in input_list:
+                    tf_shape = item.copy()
+                    tf_shape = permute_nchw_to_nhwc(tf_shape, use_new_frontend)
+                    if len(tf_inputs) == 0:
+                        tf_inputs.append(tf.compat.v1.placeholder(tf.float32, tf_shape, 'Input'))
+                    else:
+                        constant_value = np.random.randint(-3, 3, item).astype(np.float32)
+                        tf_inputs.append(tf.constant(constant_value))
+            else:
+                tf_shape = input_list.copy()
+                if len(tf_inputs) == 0:
+                    tf_inputs.append(tf.compat.v1.placeholder(tf.float32, tf_shape, 'Input'))
+                else:
+                    constant_value = np.random.randint(-3, 3, tf_shape).astype(np.float32)
+                    tf_inputs.append(tf.constant(constant_value))
```

Comment:
a lot of redundant logic. I propose to like this:

```
tf_inputs = []
for ind, input_shape in enumerate(input_shapes):
    tf_inputs.append(tf.compat.v1.placeholder(tf.float32, input_shape, 'input_{}'.format(ind)))
...
```

Reply:
Please simplify.

Reply:
Done